card/mod/carrierwave/set/type/image.rb

```diff
@@ -88,8 +88,17 @@ def new_image action
 end
 
 format :email_html do
-  view :core do |args|
-    image_tag args[:inline_attachment_url].call(card.attachment.path)
+  view :source do
+    handle_source do |source|
+      image_tag card_url(source)
+    end
+  end
+
+  view :core do
+    url_generator = voo.closest_live_option(:inline_attachment_url)
```

Comment:
Both can happen. It depends on the view used for the nested image. Because of `contextual_content` the format is usually wrong